allennlp/training/metrics/bleu.py

```diff
@@ -88,13 +74,16 @@ def _get_modified_precision_counts(
         count of that ngram in the reference sentence. The denominator is just
         the total count of predicted ngrams.
         """
+        # TODO: fix not being able to import this normally
+        from allennlp.training.util import ngrams
```

Comment:
What is this? A problem with circular imports?

Reply:
It shows up as a generic ImportError, but it looks like a circular important. I'll keep looking into it.